.k8s_ci.Dockerfile

```diff
@@ -166,7 +166,7 @@ RUN set -xe; \
 
 # Create Galaxy user, group, directory; chown
 RUN set -xe; \
-      adduser --system --group $GALAXY_USER \
+      adduser --system --group --uid 101 $GALAXY_USER \
```

Comment:
Should we set something like $GALAXY_UID at the top of the file where $GALAXY_USER is set?

Reply:
This would be great, I think this could be a follow up PR if it is important to either of you!